docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,151 @@
+
+The process for deploying a Fabric network is difficult and presumes a high degree of expertise in database management and cryptographic methods (especially x.509 certificates). If you are a smart contract or application developer, you should not need this level of expertise in deploying a production level Fabric network. However, you might need to be aware how networks are deployed in ordering to develop effective smart contracts and applications.
+
+If all you need is a development environment to test chaincode, smart contracts, and applications against, we have a [Test Network](test_network.html) that can be deployed easily to a local environment. It includes two organizations, each owning one peer, and a single ordering service organization that owns a single ordering node.
+
+## Step one: Decide on your network configuration
+
+The structure of a blockchain network must be dictated by the use case. These fundamental business decisions will vary according to your use case, but let's consider a few scenarios.
+
+In contrast to development environments or proofs of concept, security, resource management, and high availability become a priority when operating in production. How many nodes do you need to satisfy high availability, and in what data centers do you wish to deploy them in to satisfy both the needs of disaster recovery and data residency? How will you ensure that your private keys and roots of trust remain secure?
+
+Some organizations might find it necessary to establish Organizational Units to create a separation between certain identities and MSPs created by a single CA. Some channels in a network might require all data to be modeled in a way [CouchDB](couchdb_as_state_database.html) can understand, while other networks, prioritizing speed, might decide that all peers will use LevelDB. Some networks might decide that [channels](channels.html) are the best way to ensure privacy and isolation for certain transactions. Others might decide that a single channel, along with [private data collections](private-data/private-data.html), better serves their need for privacy. Different users might also make different decisions about their container orchestration, creating separate containers for their peer process, logging for the peer, CouchDB, gRPC communications, and chaincode, while other users might decide to combine some of these processes. The new [external chaincode service](cc_service.html) will also be implemented by some users and not others.
```

Comment:
This is hitting me with a lot of information quite early. Some of the information, such as private data collections, is something that users may adjust after the network is up. I think it should be focused on 1) production or not or 2) what type of CA and organizational structure they want as a result.